lib/index.js

```diff
@@ -80,6 +86,13 @@ Optimizely.prototype.track = function(track) {
   var revenue = track.revenue();
   if (revenue) payload.revenue = revenue *= 100;
   push(['trackEvent', track.event(), payload]);
+
+  var optimizelyClientInstance = window.optimizelyClientInstance;
+  if (optimizelyClientInstance && optimizelyClientInstance.track) {
+    var attributes = track.properties();
+    var userId = attributes.userId;
```

Comment:
> Basically, if you call .identify() anywhere before a .track() call, we store the userId in your cookie and all subsequent analytics.js calls like .page(), .track(), etc. will all have the userId automatically attached to the event.

Yes I passed in the user id to the `identify()` call. You can see it here in my gist: https://gist.github.com/mikeng13/0aac09f578cded939ab36031af7b48bf

I put a breakpoint inside of the `track` call of the integration and I can see that if I print out `this.analytics.user().id()` it'll show me my user id. But `track.userId()` just returns undefined. 

> Hmm so you guys have your own concept of anonymousId? Does your optimizelyClientInstance have an API that gives you read access to that value? Because maybe you could do

Our SDK instance is completely stateless so it doesn't keep track of users at all. For our customers using the integration, they might have something like a visitorId which doesn't necessarily correlate to a logged in user but they are just identifying the user for the purpose of calling our APIs for traffic splitting. 

I think using `track.userId()` would work fine but I wasn't able to get the userId populated as stated above. Could it be a bug?